react-ui/src/components/LogInForm.test.js

```diff
@@ -0,0 +1,230 @@
+    const password = getByPlaceholderText('password');
+    const submit = getByDisplayValue("Log In!");
+
+
+    // Act
+    fireEvent.change(email, { target: { value: 'joshuabrooks@gmail.com' } });
+    fireEvent.change(password, { target: { value: 'j1234' } });
+
+    const resp = { data: true };
+    await axios.get.mockImplementation(() => Promise.resolve(resp));
+    rerender(<BrowserRouter>
+      <LogInForm loginParams={testWorker} />
+    </BrowserRouter>);
+
+    fireEvent.click(submit);
+      
+    //Assert
+    rerender(<BrowserRouter>
+      <Route exact path="/login" render={props => (<Redirect to={`/dashboard/${'joshuabrooks@gmail.com'}/${'worker'}/${'Joshua'}/${8000000}/${'eeJAQr3wEC6CJZROFJTY'}`} Component={Dashboard} />)}></Route>
+    </BrowserRouter>)
+    expect(queryByPlaceholderText('email')).toBeNull();
+  });
+
+  /*  test('testing valid email and password with set changes', async () => {
+     //Arrange
+     const testRenderer = TestRenderer.create(
+       <BrowserRouter>
+         <LogInForm />
+       </BrowserRouter>);
+     const testWorker = [{
+       accessToken: "eeJAQr3wEC6CJZROFJTY",
+       email: "joshuabrooks@gmail.com",
+       firstName: "Joshua",
+       lastName: "Brooks",
+       phone: "+15191234567",
+       specialization: "Masters in Social Work",
+       type: "Guidance Counselor",
+       userType: "worker",
+       workerId: 8000000
+     }]
+ 
+     const expectedArg = "Invalid Email or Password";
+     const resp = { data:true }
+     axios.get.mockImplementation(() => Promise.resolve(resp));
+     window.alert = jest.fn();
+     // Act
+ 
+     const testInstance = testRenderer.root;
+     ReactDOM.createPortal = node => node;
+     testInstance.findByProps({ placeholder: "email" }).props.onChange({ target: { value: "joshuabrooks@gmail.com" } })
+     testInstance.findByProps({ placeholder: "password" }).props.onChange({ target: { value: "post" } })
+     testInstance.findByProps({ className: "LogInForm" }).props.onSubmit({ preventDefault: () => { } })
+ 
+     testInstance.instance({email: testRenderer.state.email}).onChange({target:{ value: "joshuabrooks@gmail.com" }} )
+      testInstance.instance.userType({target:{ value: "worker" }})
+     testInstance.instance.firstName({target:{ value: "Joshua" }})
+     testInstance.instance.workerId({target:{ value: 8000000 }})
+     testInstance.instance.accessToken({target:{ value:"eeJAQr3wEC6CJZROFJTY"}})
+     testInstance.instance.islogged({target:{value:true}}) 
+     //Assert
+     expect(window.alert).toHaveBeenCalledWith(expectedArg);
+ 
+   }); */
+
+  test('testing inputting email and password', () => {
+    //Arrange
+    const dom = render(
+      <BrowserRouter>
+        <LogInForm />
+      </BrowserRouter>);
+    const email = getByPlaceholderText(dom.container, 'email');
+    const password = getByPlaceholderText(dom.container, 'password');
+    const Email = 'joshuabrooks@gmail.com'
```

Comment:
Hi Kyle,
I would suggest renaming these variables to be `testEmail` or `userEmail` as `Email` is a special word in JavaScript (as shown by the orange colouring).

Reply:
variable has been renamed to userEmail